README.md

```diff
@@ -663,14 +671,14 @@ If it is set to `Hash`, you will need to reference the `HASH_BY_INDEX` dictionar
 
 ## Casper Event Standard
 
-When the `CES` events mode is enabled during contract installation, the operations that make changes on the tokens are recorded in the `__events` dictionary. Such event entries can be observed via a node's Server Side Events stream or querying the dictionary at any time using the RPC interface.
+`CES` is an option within the `EventsMode` modality that determines how changes to tokens issued by the contract instance will be recorded. Any changes are recorded in the `__events` dictionary and can be observed via a node's Server Side Events stream. They may also be viewed by querying the dictionary at any time using the JSON-RPC interface.
 
-The emitted events are encoded according to the [Casper Event Standard](https://github.com/make-software/casper-event-standard), and the schema can be known by an observer reading the `__events_schema` contract named key.
+The emitted events are encoded according to the [Casper Event Standard](https://github.com/make-software/casper-event-standard), and the schema is visible to an observer reading the `__events_schema` contract named key.
 
-For this CEP-78 reference implementation in particular, the events schema is the following:
+For this CEP-78 reference implementation, the events schema is as follows:
 
-| Event name      | Included values and type                                   |
-|-----------------|----------------------------------------------------------------------|
+| Event name      | Included values and type                                             |
+| --------------- | -------------------------------------------------------------------- |
```

Comment:
`token_ids` should be removed from the `ApprovalForAll` event property list

Reply:
Separately, the `token_id` should be of type `String`. `Any` type isn't supported by the CES because of decoding issues.

Reply:
@ihor 
> Is that correct that at the moment there will be no possibility to revoke approval for all and that it will be added later?

There is a possibility to revoke an operator but this was not correctly tested so I added a test below the approve_for_all test to test if revocation as operator occurred correctly.

https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/181/commits/4cbda35b24509c2e0507c1f9a5271c3c641fa530